tests/imposter/private-api.yaml

```diff
@@ -244,11 +224,11 @@ components:
               example: "2020/2021"
     Answer:
```

Comment:
I think above this, the `Question` response also needs updating - it should be `questionKey` (the code is correct)